Source/WebCore/dom/Document.cpp

```diff
@@ -6259,7 +6259,7 @@ String Document::originIdentifierForPasteboard() const
     if (origin != "null")
         return origin;
     if (!m_uniqueIdentifier)
-        m_uniqueIdentifier = "null:" + createVersion4UUIDString();
+        m_uniqueIdentifier = makeString("null:", UUID::createVersion4());
```

Comment:
We did not add `_s` here. Not thrilled that we have a mix of both idioms.

Reply:
Added.